packages/components/psammead-paragraph/README.md

```diff
@@ -14,11 +16,18 @@ It uses `@bbc/psammead-styles` for colours and font family and `@bbc/gel-foundat
 
 | Argument  | Type | Required | Default | Example |
 | --------- | ---- | -------- | ------- | ------- |
-| Script    | object | yes | latin | { canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, }|
+| Script    | object | no | latin | { canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, }|
 
 
 ## Usage
 
+```jsx
+import Paragraph from '@bbc/psammead-paragraph';
+
+const WrappingComponent = () => <Paragraph>Text here</Paragraph>;
+```
+
+When passing `script` prop:
```

Comment:
Could you please swap these examples around and clarify that this example is the recommended usage. 

That the one without the script should only be used when nesting the paragraph.

Reply:
I'm personally not convinced the ideal use case is to pass the script. We should pass the script where it is needed but the default behaviour of a browser is that the element will inherit CSS. By suggesting we explicitly pass `script` with every `<p>` just feel like a good solution IMO

Reply:
I think it should be the default usage, since otherwise you don't get any `font-size` or `line-height` defined for the paragraph and so the component no longer follows GEL Typography specs, when we've stated clearly that our `psammead` components are all GEL Foundations-compliant.

Reply:
Ahhh good point, in terms of GEL compliance as an individual component the default should be with a script. Will update